### ISSUE TITLE
seaf-fsck-sh --export fails without databases

### ISSUE THREAD

**Issue by silberzwiebel** [haiwen/seafile#1621]
Recently, I suffered a server crash and unfortunately forgot to backup seafiles mysql databases (see https://forum.seafile-server.org/t/restore-existing-seafile-data-after-server-crash/4806/).

I discovered the possibility to restore the data on the server without the databases as outlined here:
http://manual.seafile.com/maintain/seafile_fsck.html#exporting-libraries-to-file-system

However, this fails if seaf-fsck.sh cannot connect to the missing databases. The export function works, if I run it like the following (without specifying -F ../conf!):

    LD_LIBRARY_PATH=seafile/lib/:seafile/lib64:$LD_LIBRARY_PATH ./seafile/bin/seaf-fsck -c ../ccnet/ -d /path/to/seafile-data/ --export /path/to/recovery/

Also, the export option is not shown when I start ./seaf-fsck -h

**Comment by killing** [comment]
Can you give the detailed error message when running seaf-fsck.sh?

**Comment by silberzwiebel** [comment]
Not exactly, sorry. I've already recovered my system.

I tried to run `./seaf-fsck.sh --export /some/path` and before that I stopped mysql.
This gives the error message:

```
SQLException: Failed to start connection pool -- Can't connect to MySQL server on '127.0.0.1' (111)
 raised in ConnectionPool_start at src/db/ConnectionPool.c:287
```

The error message is quite similar if the database is missing. Something like "can't connect to database seafile-data" if I remember correctly.
Maybe I should mention that I'm running seafile on a raspberry pi.
